inductiva/simulators/simulator.py

```diff
@@ -54,6 +55,19 @@ def _setup_input_dir(self, input_dir: types.Path):
                 f"The provided path (\"{input_dir}\") is not a directory.")
         return input_dir
 
+    def _commands_to_json(self, kwargs):
+        if not "commands" in kwargs:
+            return kwargs
+
+        def to_cmd(x):
+            if isinstance(x, inductiva.commands.Command):
+                return x
+            return inductiva.commands.Command(x)
+
+        commands = [to_cmd(cmd).to_json() for cmd in kwargs["commands"]]
```

Comment:
Then, here would just be:

```suggestion
        commands = [to_cmd_json(cmd) for cmd in kwargs["commands"]]
```